apps/frontend/src/libs/enums/svg-icons.enum.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/// <reference types="vite-plugin-svgr/client" />
+
+import accessIcon from "../../assets/images/icons/access.svg?react";
+import analyticsIcon from "../../assets/images/icons/analytics.svg?react";
+import contributorsIcon from "../../assets/images/icons/contributors.svg?react";
+import projectIcon from "../../assets/images/icons/project.svg?react";
```

Comment:
you need to create a Icon component instead

Reply:
Take a look at this comment  and PR https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/37#discussion_r1723147909